src/styles/card__code.css

```
@@ -0,0 +1,13 @@
+/* #region card__code */
+.card__code {
+  align-self: flex-start;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  color: #616070;
+}
+
+.card__code--position {
+  margin: 0 0 16px;
```

just margin-bottom. Check similar cases
```suggestion
  margin-bottom: 16px;
```